operators/persistent-homes.yml

```diff
@@ -0,0 +1,13 @@
+- type: replace
+  path: /instance_groups/name=jumpbox/jobs/name=user_add/properties/persistent_homes?
+  value: true
+
+- type: replace
+  path: /instance_groups/name=jumpbox/persistent_disk_pool?
```

Comment:
needs to be under properties.

Reply:
argh, sorry about that. Fixed.

Reply:
Actually, not sure what you mean. `persistent_disk_pool` isn't part of properties.

Reply:
But I was missing `disk_pools` in the base jumpbox.yml